task/hiffy/src/common.rs

```diff
@@ -729,6 +731,56 @@ pub(crate) fn sprot_block_size(
     Ok(4)
 }
 
+#[cfg(feature = "sprot")]
+pub(crate) fn sprot_reset_component(
+    stack: &[Option<u32>],
+    data: &[u8],
+    _rval: &mut [u8],
+) -> Result<usize, Failure> {
```

Comment:
Adding a `use drv_update_api::ResetIntent` at the top of this function's body would make things below a lot more concise.